ssl/ssl_ciph.c

```diff
@@ -92,6 +96,7 @@ static const ssl_cipher_table ssl_cipher_table_kx[] = {
     {SSL_kPSK,      NID_kx_psk},
     {SSL_kSRP,      NID_kx_srp},
     {SSL_kGOST,     NID_kx_gost},
+    {SSL_kGOST18,   NID_kx_gost},/* FIXME beldmit */
```

Comment:
I will update it before merge to avoid reading in process.

Reply:
what?

Reply:
I want to avoid merge conflicts in obj* files, so don't introduce a separate NID until possible.

Reply:
Is now the time to add this?

Reply:
Done. Thanks!